### PR TITLE
Ported allies03b to lua

### PR DESCRIPTION
**Pull request by abcdefg30**

### CODE REVIEW

**Comment by Mailaender** [comment]
:+1: Works well and the secondary objectives give it a good replay value (I achieved discovered none at first try).

File: mods/ra/maps/allies-03b/allies03b.lua [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+	Camera.Position = Tanya.CenterPosition
+
+	InsertionHeli.Wait(DateTime.Seconds(2))
+	InsertionHeli.Move(InsertionHeliExit.Location)
+	InsertionHeli.Destroy()
+
+	Trigger.OnKilled(Tanya, function() player.MarkFailedObjective(TanyaSurvive) end)
+end
+
+SetupTopLeftIsland = function()
```
Comment (obrakmann) [inline mods/ra/maps/allies-03b/allies03b.lua]: *cough*TopRight*cough* :)

**Comment by obrakmann** [comment]
Well done again, :+1: (when you've fixed the name).

**Comment by abcdefg30** [comment]
Updated.

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/52218dfec5bdf256277bc8759262169511797845%5E...52218dfec5bdf256277bc8759262169511797845)

**Pull request merged**